tests/test_api/test_ai/test_priority_algorithm/test_priority/_data.py

```diff
@@ -0,0 +1,81 @@
+from typing import List
+
+from api.models.enums import Relationship
+from api.models.student import Student
+
+
+def create_neutral(_id: int):
+    return Student(_id=_id)
+
+
+def create_friend(_id: int, friend_ids: List[int]):
+    return Student(
+        _id=_id, relationships={_: Relationship.FRIEND for _ in friend_ids if _ != _id}
+    )
+
+
+def create_enemy(_id: int, enemy_ids: List[int]):
+    return Student(
+        _id=_id, relationships={_: Relationship.ENEMY for _ in enemy_ids if _ != _id}
+    )
+
+
+def create_social_students(
+    num_friends=None,
+    num_enemies=None,
+    num_neutrals=None,
+    num_fans=None,
+    num_haters=None,
+    num_outcasts=None,
+) -> List[Student]:
+    """
+    - friends (in a clique with each other friend)
+    - enemies (mutual enemies with everyone considered an enemy)
+    - neutral (no preferences*)
+    - fans (they like everyone in their group, but no reciprocity)
+    - haters (they hate everyone in their group but no reciprocity)
+    - outcasts (they hate everyone in their group and everyone hates them back)
+
+    * because neutral people will still also dislike outcasts because of the naive way it's programmed
+    """
+    students = []
+
+    if num_friends:
+        start_id = len(students) + 1
```

Comment:
ids should be 0 indexed

Reply:
Oh wait why?

Reply:
Usually, when I make students for tests I make their ids start at 1

Reply:
Oh, I always use 0. I guess it doesn't really matter, just thought I'd bring it up